src/m365/commands/login.ts

```diff
@@ -143,6 +152,14 @@ class LoginCommand extends Command {
       {
         option: '--thumbprint [thumbprint]',
         description: 'Certificate thumbprint. Required when authType is set to certificate'
+      },
+      {
+        option: '--appId [appId]',
+        description: 'App ID of the Azure AD application to use for authentication. If not specified, use the app specified in the CLIMICROSOFT365_AADAPPID environment variable. If the environment variable is not defined, use the multitenant PnP Management Shell app'
+      },
+      {
+        option: '--tenant [tenant]',
```

Comment:
As this is option accepts a tenant identifier, I think the name should reflect this and should be called `tenantId` instead.

Reply:
I always use "clientId" and "tenantId" in my own code, probably coming from way back when this was the terminology. appId feels unnatural to me, but it is the new thing 😃.
Same for tenant, I feel like it needs to be tenantId but it supports common/organization and <yourtenant>.onmicrosoft.com as values too. Not sure if tenantId would be matching the possibilities.

Reply:
It's not quite tenant ID, since you can specify `common` or `organization` as valid value as well. Also, internally in our code we use `tenantId` for the actual ID of the tenant with which we're communicating. We could rename this option to `tenantId` but then in our code we'd do assignment such as `auth.service.tenant = args.options.tenantId` which could be confusing, because we also have `auth.service.tenantId` which is the actual ID retrieved from the API rather than the value passed by the user.

Reply:
Hey @YannickRe, @garrytrinder, are you ok with keeping `tenant` as is or are there other arguments for using a different name that we should consider?

Reply:
Fine for me, as it allows more options than just an id.

Reply:
Fine for me, thanks for the explanation @waldekmastykarz 👍🏻